documents/FAQ.md

```diff
@@ -1,5 +1,16 @@
 # Frequently Asked Questions
 
+*__Jump To:__*
+* [Where should I start](#where-should-i-start)
+* [How do I enable logging](#how-do-i-enable-logging)
+* [I keep getting AWS_ERROR_MQTT_UNEXPECTED_HANGUP](#i-keep-getting-aws_error_mqtt_unexpected_hangup)
+* [I am experiencing deadlocks](#i-am-experiencing-deadlocks)
```

Comment:
deadlocks related to waiting in callbacks is not relevant to javascript

Reply:
thanks, removed that section